src/app/components/MediaLoader/index.tsx

```diff
@@ -67,9 +144,22 @@ type Props = {
 
 const MediaLoader = ({ blocks, className }: Props) => {
   const [isPlaceholder, setIsPlaceholder] = useState(true);
-  const { id, pageType, counterName, statsDestination, service, isAmp } =
-    useContext(RequestContext);
   const { lang, translations } = useContext(ServiceContext);
+  const { enabled: adsEnabled } = useToggle('ads');
+  const location = useLocation();
```

Comment:
Don't _think_ so, but its a good callout to test it. We changed the original `useLocation` hook awhile ago to be framework agnostic since it was tied to `react-router` before. Now it just uses the `window` object to get the `window.location`.